.github/workflows/job_build_windows.yml

```diff
@@ -82,6 +82,12 @@ jobs:
           pip-cache-path: ${{ env.PIP_CACHE_PATH }}
           should-setup-pip-paths: 'true'
           self-hosted-runner: 'true'
+      
+      - name: Setup NodeJS
+        if: ${{ fromJSON(inputs.affected-components).JS_API }}
+        uses: actions/setup-node@1d0ff469b7ec7b3cb9d8673fde0c81c44821de2a # v4.2.0
+        with:
+          node-version: 21
```

Comment:
I suggest putting it in the file-level `env` like `PYTHON_VERSION`.

Reply:
Done